rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -323,20 +348,53 @@ def _label_id_index_mapping(
     def _invert_mapping(mapping: Dict) -> Dict:
         return {value: key for key, value in mapping.items()}
 
-    def _tag_id_index_mapping(self, training_data: TrainingData) -> Dict[Text, int]:
-        """Create tag_id dictionary"""
+    def _create_crf_layers(self, training_data: TrainingData) -> List[CRFLayer]:
```

Comment:
same here I wouldn't call it create layers, we use `prepare_layers` in the model to create tf layers